src/transformer-components/DataDrivenTransformer.tsx

```diff
@@ -359,23 +361,28 @@ const DataDrivenTransformer = (props: DDTransformerProps): ReactElement => {
 
   return (
     <>
-      <Popover
-        icon={<InfoIcon htmlColor="#72bfca" fontSize="small" />}
-        tooltip={`More Info on ${base}`}
-        innerContent={
-          <>
-            <p>{splitIntoParagraphs(info.summary)}</p>
-            <p>
-              <b>Consumes: </b>
-              {splitIntoParagraphs(info.consumes)}
-            </p>
-            <p>
-              <b>Produces: </b>
-              {splitIntoParagraphs(info.produces)}
-            </p>
-          </>
-        }
-      />
+      {/* Only render info icon if NOT a saved transformation. */}
+      {saveData ? (
+        <></>
+      ) : (
```

Comment:
I'm pretty sure the following is idiomatic in React:
```diff
- {condition ? <Element /> : <></>}
+ {condition && <Element />}
```
So maybe this should be 
```jsx
{!saveData && <Popover ... />
```
Although I think the version you currently have is more readable. I'm not sure which should win out, convention or readability. @stardust66 thoughts?

Reply:
I'm okay with either though I see why the latter is conventional, it seems to leave out a lot of clutter.